### PR TITLE
Edits for celery branch

### PR DESCRIPTION
**Pull request by rusko124**
Edits:
- Merge with `master` branch
- Reworked docker-compose for dev (`./docker/start.sh up`)
- Added celery container for balenaCloud #1080
- Split all dockerfiles to multiple dockerfiles(all in `./docker`) #1124 

Tests:
- Backup and upload in balena version: worked 
balena_finished
celery

### CODE REVIEW

File: bin/install.sh [hunk 1]
```diff
@@ -57,27 +57,20 @@ EOF
     echo && read -p "Would you like to use the development branch? You will get the latest features, but things may break. (y/N)" -n 1 -r -s DEV && echo
     if [ "$DEV" != 'y'  ]; then
       export DOCKER_TAG="production"
+      echo "Screenly OSE version: Production" > ~/OSE_version.md
```
Comment (vpetersson) [inline bin/install.sh]: Production isn't a version. it's a channel. The version is the hash.

File: docker-compose.yml [hunk 2]
```diff
@@ -1,39 +1,65 @@
-version: "3"
+version: "2"
```
Comment (vpetersson) [inline docker-compose.yml]: Why are we downgrading?

Comment (rusko124) [reply docker-compose.yml]: Balena is work with this version.
`docker-compose.dev.yml` uses `version: 3`

Comment (vpetersson) [reply docker-compose.yml]: Got it. Fair.

File: bin/install.sh (hunk 1 above)
Comment (rusko124) [reply bin/install.sh]: Got it, then we need to edit this point in the master too.

Comment (vpetersson) [reply bin/install.sh]: Yep. If that's from master then we need to fix it there too.

**Pull request merged**